lib/graphql/client.ex

```diff
@@ -71,4 +100,11 @@ defmodule BridgeEx.Graphql.Client do
 
   def format_response(response) when is_map(response), do: Utils.normalize_inner_fields(response)
   def format_response(response), do: response
+
+  defp log_options do
+    Application.get_env(:bridge_ex, :log_options,
+      log_query_on_error: false,
+      log_response_on_error: false
+    )
```

Comment:
As a aside note, my only concern is about the library guideline [avoid application configuration](https://hexdocs.pm/elixir/1.13/library-guidelines.html#avoid-application-configuration). But it's not critical, just keep it in mind.

Reply:
I didn't know about this behaviour and it could be a footgun... I'll open an issue to discuss it, thanks 😊